### ISSUE TITLE
Should not the same release banner link to the same page?

### ISSUE THREAD

**Issue by Bjwebb** [open-contracting/standard-collaborator#31]
e.g. should the banner on http://ocds.open-contracting.org/standard/r/1__0__RC/en/schema/codelists/ link to http://ocds.open-contracting.org/standard/r/1__0__0/en/schema/codelists/ rather than http://ocds.open-contracting.org/standard/r/1__0__0/en/

Removed pages may be an issue for this suggestion.

**Comment by timgdavies** [comment]
Yes - that would be good, with, as you mentioned, the issue of removed pages presenting a small challenge.

Perhaps: 

* If the page exists in the new version, link to it;
* If it does not, link to the root of the version;
